Livecheckables/pipes-sh.rb

```diff
@@ -0,0 +1,6 @@
+class PipesSh
+  livecheck do
+    url :stable
```

Comment:
```suggestion
    url :head
```

The stable URL is a GitHub archive, which gets converted into the Git repo URL and is identical to the formula's `head` URL.